share/p2p/shrexsub/pubsub.go

```diff
@@ -15,9 +15,9 @@ import (
 
 var log = logging.Logger("shrex-sub")
 
-// pubsubTopic hardcodes the name of the EDS floodsub topic with the provided suffix.
-func pubsubTopic(suffix string) string {
-	return fmt.Sprintf("/eds-sub/v0.0.1/%s", suffix)
+// PubsubTopic hardcodes the name of the EDS floodsub topic with the provided networkID.
+func PubsubTopicID(networkID string) string {
```

Comment:
Not necessary to make it public, as we there is no scoring for FloodSub

Reply:
okay